OloEngine/src/OloEngine/Scripting/VisualScript/VisualScriptSerializer.cpp

```
@@ -0,0 +1,380 @@
+            for (const YAML::Node& entry : functions)
+            {
+                if (VisualScriptGraph function; DeserializeGraph(entry, function))
+                {
+                    asset.m_Functions.push_back(std::move(function));
+                }
+            }
+        }
+        return true;
+    }
+
+    bool VisualScriptSerializer::Serialize(const VisualScriptAsset& asset, const std::filesystem::path& path)
```

Cppcheck (2.21.0)

[style] The function 'Serialize' is never used.

(unusedFunction)